kafka_utils/kafka_consumer_manager/commands/offset_get.py

```diff
@@ -140,7 +181,8 @@ def get_offsets(cls, client, group, topics_dict, storage):
     @classmethod
     def print_output(cls, consumer_offsets_metadata, watermark_filter):
         for topic, metadata_tuples in consumer_offsets_metadata.iteritems():
-            print ("Topic Name: {topic}".format(topic=topic))
+            diff_sum = sum([t.highmark - t.current for t in metadata_tuples])
+            print ("Topic Name: {topic}  Total difference: {diff}".format(topic=topic, diff=diff_sum))
```

Comment:
That should be "Distance", since it's the same word used in the rest of the output. I'll change it. I think adding "offset" would be redundant since it's obvious from the rest of the output.

New output:
```
Topic Name: topic0  Total Distance: 167
        Partition ID: 0
                High Watermark: 256769105
                Low Watermark: 256456931
                Current Offset: 256769079
                Offset Distance: 26
                Percentage Distance: 0.0%
        Partition ID: 1
                High Watermark: 25124601
                Low Watermark: 24822641
                Current Offset: 25124460
                Offset Distance: 141
                Percentage Distance: 0.0%
Topic Name: topic1  Total Distance: 348373
        Partition ID: 0
                High Watermark: 46817713
                Low Watermark: 43965920
                Current Offset: 46469340
                Offset Distance: 348373
                Percentage Distance: 0.74%
```